src/ec2/explorer/ec2InstanceNode.ts

```diff
@@ -45,7 +45,7 @@ export class Ec2InstanceNode extends AWSTreeNodeBase implements AWSResourceNode
     }
 
     public isPending(): boolean {
-        return this.getStatus() != 'running' && this.getStatus() != 'stopped'
+        return this.getStatus() == 'pending'
```

Comment:
usually statuses are uppercase, are we lowering it somewhere?
```suggestion
        return this.getStatus() == 'PENDING'
```

Reply:
Believe the API does return lower case here: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/EC2.html#describeInstanceStatus-property. Not sure why it's not consistent because I noticed it is upper case for CodeCatalyst.